polynote-kernel/src/main/scala/polynote/kernel/remote/transport.scala

```diff
@@ -151,22 +145,25 @@ object SocketTransportServer {
   ): TaskB[SocketTransportServer] = for {
     closed   <- Promise.make[Throwable, Unit]
     channels <- selectChannels(channel1, channel2, server.getLocalAddress.asInstanceOf[InetSocketAddress])
-    _        <- channel1.awaitClosed.to(closed).fork
-    _        <- channel2.awaitClosed.to(closed).fork
+    _        <- channel1.awaitClosed.to(closed).forkDaemon
+    _        <- channel2.awaitClosed.to(closed).forkDaemon
   } yield new SocketTransportServer(server, channels, process, closed)
 }
 
 class SocketTransportClient private (channels: SocketTransport.Channels, closed: Promise[Throwable, Unit]) extends TransportClient {
 
   def logError(fn: Cause[Throwable] => ZIO[Logging, Nothing, Unit]): TaskB ~> TaskB = new ~>[TaskB, TaskB] {
-    override def apply[A](fa: TaskB[A]): TaskB[A] = fa.onError(fn)
+    override def apply[A](fa: TaskB[A]): TaskB[A] = fa.onError {
+      case cause if cause.interruptedOnly => ZIO.unit
```

Comment:
What does this change do? If we interrupted the effect it won't log it as an error? Or something else?

Reply:
Yup!

Reply:
Nice!